docs/config.md

```diff
@@ -142,8 +148,30 @@ model_names: <comma-delimited-string|list|model>
 
 # Disables automatic config search
 [ run_config_search_disable: <bool> | default: false ]
+
+# Specify path to config yaml file
+[ config_file: <string> ]
```

Comment:
Can you also add the `triton_docker_image`?

Reply:
It is already present. The first argument after the perf analyzer related arguments

Reply:
I don't remember adding it myself

Reply:
Yeah, I added it in my PR. I forgot about it :)